internal/cmd/operator-sdk/generate/bundle/cmd.go

```diff
@@ -129,6 +130,8 @@ func (c *bundleCmd) addFlagsTo(fs *pflag.FlagSet) {
 		"This option can only be used if --deploy-dir is set")
 	fs.StringVar(&c.kustomizeDir, "kustomize-dir", filepath.Join("config", "manifests"),
 		"Directory containing kustomize bases in a \"bases\" dir and a kustomization.yaml for operator-framework manifests")
+	fs.StringVar(&c.customAnno, "custom-annotations", filepath.Join("config", "metadata"),
+		"Directory containing custom annotations for operator bundle metadata")
```

Comment:
I think we should not create a new dir / layout

Reply:
See: https://github.com/operator-framework/operator-sdk/pull/5560#issuecomment-1049202882